app/views/applicationcheck/contactDetails.scala.html

```diff
@@ -23,7 +23,7 @@
 @titleError = { @if(form.errors.nonEmpty) {Error: } else {}}
 
 @devMain(
-    title = titleError + "Who is your application review contact?",
+    title = titleError + "Who to contact about your application",
```

Comment:
You need to use @pageHeading to save on duplication IIRC

Reply:
Yes